okio/build.gradle

```diff
@@ -66,45 +100,83 @@ kotlin {
       dependsOn nonJvmMain
       dependencies {
         api deps.kotlin.stdLib.js
+        implementation("org.jetbrains.kotlinx:kotlinx-nodejs:0.0.7")
```

Comment:
Does this block Okio's APIs from going stable? Perhaps it would be better to define external functions for the things in use and drop this dependency.

Reply:
Yeah, lemme add that work to the queue.